examples/resources/grafana_data_source_config_lbac_rules/resource.tf

```diff
@@ -0,0 +1,33 @@
+resource "grafana_team" "team" {
+  name = "Team Name"
+}
+
+resource "grafana_data_source" "test" {
+  type                = "loki"
+  name                = "loki-from-terraform"
+  url                 = "https://mylokiurl.net"
+  basic_auth_enabled  = true
+  basic_auth_username = "username"
+
+  json_data_encoded = jsonencode({
+    authType = "default"
+    ## basicAuthPassword = "<>"
+  })
+}
+
+# TODO: add back 5th of dec
```

Comment:
This commended out code block also appears in [the docs](https://github.com/grafana/terraform-provider-grafana/pull/1797/files#diff-699c776ea911e1496728900cde3d15c92f34800e5f2701eec97295020257f320R43-R57), so we should really remember to uncomment the code before we do the next Terraform release.

Or maybe just track this as an action item in an issue rather than having it here.

Reply:
il track it in an item instead 👍

Reply:
https://github.com/grafana/identity-access-team/issues/1047